notebooks/logical_qubit_extended_memory_experiment.ipynb

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
<p>The code generating the logical memory circuits looks good to me. Some ideas on the simulation part to gain more confidence on the correctness of implementation:</p><p> </p><ol><li>We can simulate for both of the X/Z logical basis to verify that the extended logical basis outperforms the regular one.</li><li>We can simulate a equal-distance square logical qubit memory experiment for comparison. If the implementation is correct, the extended logical basis should outperform that in the square case. And the regular logical basis should perform worse than that in the square case(there are more error paths for it to fail).</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/QCHackers/tqec/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I will implement these validation in a follow-up PR, thanks for the remarks!